ratpack-handlebars/src/main/java/ratpack/handlebars/HandlebarsModule.java

```diff
@@ -94,6 +97,10 @@
 
     private String templatesSuffix = ".hbs";
 
+    private String startDelimiter = "{{";
```

Comment:
I think we should prob use `com.github.jknack.handlebars.DELIM_START` rather than hard coding to `{{`.  That way we're protected should they change the default.